packages/edit-site/src/components/sidebar/global-styles-v2/views/colors.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import { useGlobalStylesNavigationContext } from '../navigation/context';
+
+export default function GlobalStylesViewColors() {
+	const { setCurrentView } = useGlobalStylesNavigationContext();
+
+	const navigateToColorsElement = () => setCurrentView( 'colors-element' );
+	const navigateToRoot = () => setCurrentView( 'root' );
+
+	return (
+		<>
+			<h3>Colors view</h3>
+			<button onClick={ navigateToRoot }>&lt; Back to Root view</button>
```

Comment:
This is just for demo purposes for now, but it already shows that we probably need general components with things like the back button to go back to previews view, the title of the view etc.

Reply:
Of course 👍